vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -43,12 +43,21 @@
 
   # Install BMO
   - name: Install Baremetal Operator
-    shell: "kustomize build /home/$USER/go/src/github.com/metal3-io/baremetal-operator/deploy/default/ | kubectl --kubeconfig=/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml apply -n {{ NAMESPACE }} -f -"
+    shell: "{{ BMOPATH }}/tools/deploy.sh {{ DEPLOY_BMO }} {{ DEPLOY_IRONIC }} {{ DEPLOY_TLS }} {{ DEPLOY_BASIC_AUTH }} {{ DEPLOY_KEEPALIVED }}"
```

Comment:
here DEPLOY_BMO should be just "true" and DEPLOY_IRONIC just "false", since you want to deploy BMO only

Reply:
Addressed in #516